From patchwork Wed Aug  7 13:58:14 2019
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Naresh Kamboju <naresh.kamboju@linaro.org>
X-Patchwork-Id: 11082209
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 53E3914E5
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed,  7 Aug 2019 13:58:24 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 45F3428A19
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed,  7 Aug 2019 13:58:24 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 3051B28A4C; Wed,  7 Aug 2019 13:58:24 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-8.0 required=2.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI
	autolearn=unavailable version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id E7F2B1FF40
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed,  7 Aug 2019 13:58:23 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1729910AbfHGN6U (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Wed, 7 Aug 2019 09:58:20 -0400
Received: from mail-wr1-f68.google.com ([209.85.221.68]:40716 "EHLO
        mail-wr1-f68.google.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1729860AbfHGN6T (ORCPT <rfc822;kvm@vger.kernel.org>);
        Wed, 7 Aug 2019 09:58:19 -0400
Received: by mail-wr1-f68.google.com with SMTP id r1so91454050wrl.7
        for <kvm@vger.kernel.org>; Wed, 07 Aug 2019 06:58:18 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=linaro.org; s=google;
        h=from:to:cc:subject:date:message-id;
        bh=/q1A4PrAblYMBCQgI2N1FaerB81pHXKu9hE/Sa+XUXA=;
        b=vVIW9zlPe5Sze48sRoyXSlGRmh5x3AQpyelKbs+7nugJJ3s7eW40oTSh4LS+0TepSH
         H28kfwS3fIVk1nXC4oYrri7e/or/i1HcprEvxcR19ur7bYuBxlTnf2+zoeMhd0tFus+l
         dhPcWRwqjlNeh2qaEFHYIj99UE/75l3L45NgGD1JuYO7Rw9ffCQktznfRf+uLUg8XcZ5
         YCTqSUTBUzwI7fRA0nt1jA3AYY1NPvZ30qp5pr402FeeWyQKFBx4FQ8qvFNCGqWLva0l
         NgM8yXtvwZqYTopT2tRhnwuEau+wHMziDjEQlrSBPIRxAITECY+Zl3mBfANfcUvIx5h0
         MSsg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:from:to:cc:subject:date:message-id;
        bh=/q1A4PrAblYMBCQgI2N1FaerB81pHXKu9hE/Sa+XUXA=;
        b=bKg/MnQO+MCYA+AaMD+6/AQiSFFP8Ll1awl96YCnOS6haZdJEzYmpEU+6Vnqk7bSNy
         +EGBJKuHffJ61ulk8mYsqHrHfD3jmX2FBe/Pqqf/7m5obCDTrV+xVES3sCjsATDReKQt
         plAJ9A0U28qT3LHU4sEl7GqOiMaqvQUQuLK0G5FV8yg2QySKTIZzfBmuW06i5WC42YB4
         2PDHkd+qg/kbzNCeKP3helHjc6FIX4SmIeIenP5Pl4pTiXLlg6yBGWsBDJjDu55byYj9
         UnpvgCgTQJM3qyMaKRkn0wLlUg8nM1EH0+ljHUMqbapL4V75PZ8SysLjMKOQX3uukP21
         DGcQ==
X-Gm-Message-State: APjAAAUZNb8Sf0FVyKeBthYlcNqLkg+lg+o+KnwgpYuLCBhXa0ATX+wh
        v2UMBInsP5QlJkVhmWb+yW0gog==
X-Google-Smtp-Source: 
 APXvYqxZjoJ4iUF7J7s+ussP5b6+gLvzvOyEPFVQwRFhxv+xdTUkl3QcOk/Eg+A0DCXIq10UbctGYw==
X-Received: by 2002:adf:b1cb:: with SMTP id
 r11mr10524405wra.328.1565186297540;
        Wed, 07 Aug 2019 06:58:17 -0700 (PDT)
Received: from hackbox2.linaro.org ([81.128.185.34])
        by smtp.gmail.com with ESMTPSA id
 n12sm2912467wrw.25.2019.08.07.06.58.16
        (version=TLS1_3 cipher=AEAD-AES256-GCM-SHA384 bits=256/256);
        Wed, 07 Aug 2019 06:58:16 -0700 (PDT)
From: Naresh Kamboju <naresh.kamboju@linaro.org>
To: pbonzini@redhat.com, linux-kernel@vger.kernel.org,
        drjones@redhat.com, sean.j.christopherson@intel.com,
        linux-kselftest@vger.kernel.org, kvm@vger.kernel.org
Cc: Naresh Kamboju <naresh.kamboju@linaro.org>
Subject: [PATCH v2] selftests: kvm: Adding config fragments
Date: Wed,  7 Aug 2019 14:58:14 +0100
Message-Id: <20190807135814.12906-1-naresh.kamboju@linaro.org>
X-Mailer: git-send-email 2.17.1
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

selftests kvm test cases need pre-required kernel configs for the test
to get pass.

Signed-off-by: Naresh Kamboju <naresh.kamboju@linaro.org>
---
 tools/testing/selftests/kvm/config | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 tools/testing/selftests/kvm/config

diff --git a/tools/testing/selftests/kvm/config b/tools/testing/selftests/kvm/config
new file mode 100644
index 000000000000..63ed533f73d6
--- /dev/null
+++ b/tools/testing/selftests/kvm/config
@@ -0,0 +1,3 @@
+CONFIG_KVM=y
+CONFIG_KVM_INTEL=y
+CONFIG_KVM_AMD=y
